src/components/TextInput.vue

```diff
@@ -48,36 +48,49 @@ export default {
 
 .floating-placeholder {
   color: #8e8e8e;
-  margin-bottom: -25px;
+  align-items: center;
```

Comment:
Something seems to be making the input box jump now.
On the home page, go to mobile style view, and click inside a box.
You'll see it jump, I think this is from the floating label.

Reply:
fixed, by deleting margin-bottom: -23px; font-size: 15px;